File: examples/AccountUpdateExample.rb
require "json"
require "dropbox-sign"

Dropbox::Sign.configure do |config|
    config.username = "YOUR_API_KEY"
    # config.access_token = "YOUR_ACCESS_TOKEN"
end

account_update_request = Dropbox::Sign::AccountUpdateRequest.new
account_update_request.callback_url = "https://www.example.com/callback"
account_update_request.locale = "en-US"

begin
    response = Dropbox::Sign::AccountApi.new.account_update(
        account_update_request,
    )

    p response
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling AccountApi#account_update: #{e}"
end

File: sandbox/php/src/AccountUpdateExample.php
<?php

namespace Dropbox\SignSandbox;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use Dropbox;

$config = Dropbox\Sign\Configuration::getDefaultConfiguration();
$config->setUsername("YOUR_API_KEY");
// $config->setAccessToken("YOUR_ACCESS_TOKEN");

$account_update_request = (new Dropbox\Sign\Model\AccountUpdateRequest())
    ->setCallbackUrl("https://www.example.com/callback")
    ->setLocale("en-US");

try {
    $response = (new Dropbox\Sign\Api\AccountApi(config: $config))->accountUpdate(
        account_update_request: $account_update_request,
    );

    print_r($response);
} catch (Dropbox\Sign\ApiException $e) {
    echo "Exception when calling AccountApi#accountUpdate: {$e->getMessage()}";
}
